Xamarin.Forms.Core/RefreshView.cs

```diff
@@ -20,7 +22,41 @@ public RefreshView()
 		}
 
 		public static readonly BindableProperty IsRefreshingProperty =
-			BindableProperty.Create(nameof(IsRefreshing), typeof(bool), typeof(RefreshView), false, BindingMode.TwoWay);
+			BindableProperty.Create(nameof(IsRefreshing), typeof(bool), typeof(RefreshView), false, BindingMode.TwoWay, coerceValue:OnCoerceIsRefreshing, propertyChanged: OnIsRefreshingPropertyChanged);
+
+		static void OnIsRefreshingPropertyChanged(BindableObject bindable, object oldValue, object newValue)
+		{
+			bool value = (bool)newValue;
+
+			if (value)
+			{
+				var resfrehView = ((RefreshView)bindable);
+				resfrehView.Refreshing?.Invoke(bindable, EventArgs.Empty);
+				if(resfrehView.Command != null)
+					resfrehView.Command.Execute(resfrehView.CommandParameter);
+			}
+		}
+
+		static object OnCoerceIsRefreshing(BindableObject bindable, object value)
```

Comment:
Should this be renamed to `OnIsRefreshingPropertyCoerced` to align with the previous PC handler?